### PR TITLE
add package APIs

### PR DESCRIPTION
**Pull request by Yiyun-Liang**

### CODE REVIEW

File: seed/seed.js [hunk 1]
```diff
@@ -69,6 +69,66 @@ let clear = function (config, done) {
 
   let q = clearQ(MYSQL_DB)
 
+  q = `CREATE DATABASE IF NOT EXISTS \`${process.env.MYSQL_DB}\`;
```
Comment (ben-z) [inline seed/seed.js]: The `clear` function is meant to be used to clear all side effects created by `create` above. Is this change necessary for the code to work?

Comment (Yiyun-Liang) [inline seed/seed.js]: This is needed for the build to work because otherwise we would have duplicated the variable q. Does it have to be q for the second one?

**Comment by ben-z** [comment]
I think we don't need to reassign `q` at all, since this function is for clearing all the changes made by `create`. If we look at [query.js](1) lines `74-76`:

```es6
let q = `DROP DATABASE IF EXISTS \`${db_name}\`;`

return q
```

that's what our first assignment of `q` in this file does:

```js
let q = clearQ(MYSQL_DB)
```

So we don't need to give q another hard_coded value.

[1]: https://github.com/PackAnts/API/blob/34cee7ef9fdfcaf45ed04460287c35334c90057f/seed/lib/query.js

**Comment by Yiyun-Liang** [comment]
So how do you think we should change it?

**Comment by ben-z** [comment]
I sent a pull request :+1:

**Comment by ben-z** [comment]
Can you write some tests for `/package` too? An example can be found in `/test/routes/user_spec.js`.

**Comment by Yiyun-Liang** [comment]
Sure I will look into that when I get back home today:)

**Comment by ben-z** [comment]
Awesome :smile:

**Comment by ben-z** [comment]
I will merge this pull request after the tests for `/package` are completed :+1:

**Comment by Yiyun-Liang** [comment]
Sure
